falcon/routing/util.py

```diff
@@ -92,7 +92,7 @@ def create_http_method_map(resource):
             ``on_post(self, req, resp)``.
 
     Returns:
-        dict: A mapping of HTTP methods to responders.
+        dict: A mapping of HTTP methods to explicitly defined resource responders.
```

Comment:
I think we had better not change the functionality of `create_http_method_map()` since that would be a breaking change and this issue is scheduled for a minor point release. Instead, what do you think about introducing a new function (say, `map_http_methods()`)? It could be used in lieu of `create_http_method_map()` at `api.py:348` and we could add a note to the docstring for `create_http_method_map()` saying that it's deprecated and will be removed in a future release (use the new methods instead).

Reply:
Right, that makes a lot of sense. Added the new method and reverted `create_http_method_map` back to its original form.